src/markup/codemirror/create.ts

```diff
@@ -151,7 +152,33 @@ export function createCodemirror(params: CreateCodemirrorParams) {
                 onScroll(event);
             },
             paste(event, editor) {
-                if (event.clipboardData && parseInsertedUrlAsImage) {
+                console.log(event.clipboardData);
```

Comment:
I suggest deleting the console logs placed for debag

Reply:
fixed